influxdb/influxdb08/dataframe_client.py

```diff
@@ -13,6 +13,16 @@
 from .client import InfluxDBClient
 
 
+def total_seconds(td):
```

Comment:
Rather than duplicating this function, can you add it as a method to the InfluxDBClient class?